### PR TITLE
Basic package structure

### PR DESCRIPTION
**Pull request by br-programmer**
#2 Solved

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: animated_book_list
+description: A Dart Package for comunity the-d-velopers
```
Comment (monster555) [inline pubspec.yaml]: "A Flutter package from the d-velopers community"?

**Review by monster555 — APPROVED** [review]
LGTM :)

**Comment by diegoveloper** [comment]
Can you remove the .DS_store file and `enums.dart` file (this is not being used).

**Review by diegoveloper — APPROVED** [review]

**Review by Adrianns — APPROVED** [review]

**Review by victormanuelfrancodev — APPROVED** [review]
👍 ok

**Pull request merged**